modules/hal_nordic/nrf_802154/serialization/platform/nrf_802154_spinel_backend_ipc.c

```diff
@@ -9,6 +9,11 @@
 #include <zephyr/device.h>
 #include <zephyr/logging/log.h>
 
+#if defined(CONFIG_SOC_NRF5340_CPUAPP)
+#include <nrf53_cpunet_mgmt.h>
+#include <hal/nrf_spu.h>
+#endif
```

Comment:
It would be good to merge this with https://github.com/zephyrproject-rtos/zephyr/blob/52af1762d16fb85246de869d8b19d2172cf70c03/modules/hal_nordic/nrf_802154/serialization/platform/nrf_802154_spinel_backend_ipc.c#L23-L25